R/shinywrappers.R

```diff
@@ -10,7 +13,7 @@ utils::globalVariables('func', add = TRUE)
 #'
 #' Note that it is generally preferable to use [createRenderFunction()] instead
 #' of `markRenderFunction()`. It essentially wraps up the user-provided
-#' expression in the `transform` function passed to it, then pases the resulting
+#' expression in the `transform` function passed to it, then passes the resulting
 #' function to `markRenderFunction()`. It also provides a simpler calling
 #' interface.
```

Comment:
```suggestion
#' interface. There may be cases where `markRenderFunction()` must be used instead of
#' [createRenderFunction()] -- for example, when the `transform` parameter of
#' [createRenderFunction()] is not flexible enough for your needs.
```